dts/bindings/misc/nxp,s32-emios.yaml

```
@@ -36,6 +36,14 @@ properties:
     description: |
       A mask for channels that have internal counter, lsb is channel 0.
 
+  gtbe-provider:
```

```suggestion
  global-timebase-provider:
```
a bit longer but less cryptic

---

I updated